### PR TITLE
Reworked player hitboxes

### PR DESCRIPTION
**Pull request by MrJackey**
## Summary
Fixes # 60, 62
The sprite and hitboxes are now separated into children to the main player object.
Moved the player center point towards the center of the sprite.

## Visualisation
5OG0HG2dJq

### CODE REVIEW

File: Assets/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -59,13 +59,15 @@ public class PlayerController : MonoBehaviour {
 	private float dashResetTimer = 0.5f;
 
 	public bool AllowControls { set => allowControls = value; }
+
+	public Collider2D GroundedCollider { set => groundedCollider = value; }
+
```
Comment (smeas) [inline Assets/Scripts/Player/PlayerController.cs]: This property is never used.

**Review by smeas — CHANGES_REQUESTED** [review]
The player prefab has the wrong ground collider assigned so you can't jump.

File: Assets/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (MrJackey) [reply Assets/Scripts/Player/PlayerController.cs]: Right, that was the first implementation. Will remove

**Review by smeas — APPROVED** [review]
+1, Awesome!

**Review by Addehman — APPROVED** [review]
wunderbar

**Pull request merged**